client/my-sites/migrate/section-migrate.jsx

```diff
@@ -365,28 +365,29 @@ class SectionMigrate extends Component {
 					{ translate( 'There was an error with your import.' ) }
 					<br />
 					{ this.state.errorMessage }
+
+					<p className="migrate__info">
+						{ translate(
+							'{{supportLink}}Contact us{{/supportLink}} so we can' +
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **14** times:
`translate( 'Or {{supportLink}}contact us{{/supportLink}} so we can figure out exactly what needs adjusting and get your site imported.' )` *ES Score: 10*